src/theme/DocItem/Layout/index.js

```diff
@@ -31,10 +34,51 @@ function useDocTOC() {
     desktop,
   };
 }
+
+
 export default function DocItemLayout({children}) {
   const docTOC = useDocTOC();
   const {metadata} = useDoc();
   const {editUrl} = metadata;
+
+  const location = useLocation();
```

Comment:
this can all be done much simpler, use the getDocusarusContext and you can get the locale. Just check its not en